scripts/NextJs/configs/next.config.js

```diff
@@ -7,18 +7,34 @@ const { withSentryConfig } = require('@sentry/nextjs');
 
 const moduleExports = {
   // Your existing module.exports
+
+  sentry: {
+    // Use `hidden-source-map` rather than `source-map` as the Webpack `devtool`
+    // for client-side builds. (This will be the default starting in
+    // `@sentry/nextjs` version 8.0.0.) See
```

Comment:
> (This will be the default starting in `@sentry/nextjs` version 8.0.0.)
 
Wondering if this is just too much information for people. Telling people this should not change how they're configuring or using stuff. IMO it's fine to just tell them in v8 to remove this option (like you wrote in the comment in https://github.com/getsentry/sentry-javascript/pull/5649). No hard opinion though.

Reply:
I also don't feel strongly about it, but it's what I've seen other tools do, and I figure it lets people make their own `TODO` if they want to. I'm inclined to leave it in.

Reply:
Sounds good!